templates/coordinator/_citizen_project_list.html.twig

```diff
@@ -0,0 +1,136 @@
+<table class="datagrid__table" style="border-spacing: 0 1em;">
+    <thead>
+        <tr>
+            <th>Titre</th>
+            <th>Organisateur</th>
+            <th>Membres</th>
+            <th>Comités en soutien</th>
+            <th>Lieu</th>
+            <th>Problème local adressé</th>
+            <th>Réponse au problème</th>
+            <th>Feuille de route</th>
+            <th>Compépetences matchées</th>
+            <th>Contenu accompagnement</th>
+            <th>Commentaire</th>
+            <th>Statut</th>
+            {% if app.request.query.get('s') == constant('AppBundle\\Entity\\CitizenProject::APPROVED') %}
+                <th>Signalement</th>
+                <th>Résolution du signalement</th>
+            {% endif %}
+        </tr>
+    </thead>
+    <tbody id="citizen-projects-list">
+        {% for project in results %}
+            {% if not project.isApproved %}
+                {% set form = forms[project.id] %}
+
+                {{ form_start(form) }}
+
+                {{ form_errors(form) }}
```

Comment:
Please indent the code between `form_start` and `form_end` as you would with regular form tags.